fe/src/api/postUserLogin.js

```diff
@@ -0,0 +1,25 @@
+// src/api/postUserLogin.js
```

Comment:
define a single file named users sau login(daca vrei sa fie doar logica pentru login) and put there all the apis for user
ambele functii le pui acolo